### PR TITLE
fix error while api network is not stable 

### PR DESCRIPTION
**Pull request by scream7**
Short Description: 

the object may get NoneType if api return none.

Fixes:
- 
- 
-

### CODE REVIEW

File: pokemongo_bot/__init__.py [hunk 1]
```diff
@@ -365,8 +365,11 @@ def item_inventory_count(self, id):
         self.api.get_player().get_inventory()
 
         inventory_req = self.api.call()
-        inventory_dict = inventory_req['responses'][
-            'GET_INVENTORY']['inventory_delta']['inventory_items']
+        try:
+            inventory_dict = inventory_req['responses'][
+                'GET_INVENTORY']['inventory_delta']['inventory_items']
+        except:TypeError:
```
Comment (TheSavior) [inline pokemongo_bot/__init__.py]: In what case would we get this TypeError?

Comment (scream7) [inline pokemongo_bot/__init__.py]: @TheSavior I got this error yesterday, it seems when call "api.call()" failed (may be due to the netwoking problems?) the "inventory_req" will get a NoneType, and thus the program will exit.

**Comment by surfaace** [comment]
could you please resolve merge issues so approval can be made?

**Comment by scream7** [comment]
@surfaace sorry for typo, I have resolved it.

**Comment by surfaace** [comment]
:+1:

**Comment by bstpierr** [comment]
This will need merged again since the cached inventory response PR.

**Comment by surfaace** [comment]
@scream7 please resolve conflict introduced with cached inventory

**Comment by scream7** [comment]
@surfaace @bstpierr It seems working.

**Pull request closed without merging**